app/scripts/controllers/createPersistentVolumeClaim.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:CreatePersistentVolumeClaimController
+ * @description
+ * # CreatePersistentVolumeClaimController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('CreatePersistentVolumeClaimController', function ($filter, $routeParams, $scope, $window, ApplicationGenerator, DataService, Navigate, ProjectsService) {
+    $scope.alerts = {};
+    $scope.projectName = $routeParams.project;
+    $scope.serviceName = $routeParams.service;
+    $scope.accessModes="ReadWriteOnce";
+    $scope.pvc = {
+      name: $scope.serviceName || ""
+    };
```

Comment:
`$scope.pvc` is not actually read anywhere. You should be able to remove it (when you remove the list services block).